config/apiserver/rbac/service_account.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: sa
+  namespace: system
```

Comment:
Add `namespace: system` here, otherwise, if someone specifies this service account as a subject in the `subjects` with namespace:
```yaml
subjects:
- kind: ServiceAccount
  name: sa
  namespace: system
```
Then this will not be modified by kustomize with the correct name & namespace of the service account